PhysicsTools/NanoAOD/python/jets_cff.py

```diff
@@ -227,12 +227,14 @@
         electronIdx1 = Var("?overlaps('electrons').size()>0?overlaps('electrons')[0].key():-1", int, doc="index of first matching electron"),
         electronIdx2 = Var("?overlaps('electrons').size()>1?overlaps('electrons')[1].key():-1", int, doc="index of second matching electron"),
         nElectrons = Var("?hasOverlaps('electrons')?overlaps('electrons').size():0", int, doc="number of electrons in the jet"),
-        btagCMVA = Var("bDiscriminator('pfCombinedMVAV2BJetTags')",float,doc="CMVA V2 btag discriminator",precision=10),
         btagDeepB = Var("bDiscriminator('pfDeepCSVJetTags:probb')+bDiscriminator('pfDeepCSVJetTags:probbb')",float,doc="DeepCSV b+bb tag discriminator",precision=10),
```

Comment:
is needed to change the btagDeepB to btagDeepBvL  ? same as you have done for the btagDeepCvL and btagDeepCvB ?

Reply:
No, the b definition remains the same. The charm changes are due to new validation procedure + recs

Reply:
@andrzejnovak What was discussed during the XPOG meeting was to store in nanoAOD the binarised scores [the ones analyses use and are calibrated]. So for btagDeepFlavB should be the "BvsL" [b+bb+blep]/[b+bb+blep+udsg]. Unless there are analyses use the raw score [which was not discussed in the meeting]

Reply:
@gouskos That is not however what's being calibrated. What is calibrated is `P(b)+P(bb)+P(lepb)`. By default btagging is b vs rest, not b vs light.

Reply:
Thanks for the clarification @andrzejnovak !